res/values/strings_tl.arb

```diff
@@ -0,0 +1,686 @@
+  "avg_savings": "Average na pagtitipid",
+  "view_all": "Tingnan lahat",
+  "active_cards": "Mga aktibong card",
+  "delete_account": "Tanggalin ang Account",
+  "cards": "Mga Card",
+  "active": "Aktibo",
+  "redeemed": "Natubos",
+  "gift_card_balance_note": "Lalabas dito ang mga gift card na may natitirang balanse",
+  "gift_card_redeemed_note": "Lalabas dito ang mga gift card na na-redeem mo",
+  "logout": "Logout",
+  "add_tip": "Magdagdag ng Tip",
+  "percentageOf": "ng ${amount}",
+  "is_percentage": "ay",
+  "search_category": "Kategorya ng paghahanap",
+  "mark_as_redeemed": "Markahan bilang natubos",
+  "more_options": "Higit pang mga pagpipilian",
+  "awaiting_payment_confirmation": "Nanghihintay ng Kumpirmasyon sa Pagbabayad",
+  "transaction_sent_notice": "Kung hindi magpapatuloy ang screen pagkatapos ng 1 minuto, tingnan ang block explorer at ang iyong email.",
+  "agree": "Sumang-ayon",
+  "in_store": "Nakatago",
+  "generating_gift_card": "Pagbuo ng Gift Card",
+  "payment_was_received": "Natanggap ang iyong bayad.",
+  "proceed_after_one_minute": "Kung ang screen ay hindi magpapatuloy pagkatapos ng 1 minuto, suriin ang iyong email.",
+  "order_id": "Order ID",
+  "gift_card_is_generated": "Nabuo ang Gift Card",
+  "open_gift_card": "Buksan ang Gift Card",
+  "contact_support": "Makipag-ugnayan sa tagatustos",
+  "gift_cards_unavailable": "Ang mga Gift Card ay magagamit lamang para bilhin gamit ng Monero, Bitcoin, at Litecoin sa ngayon",
+  "introducing_cake_pay": "Pagpapakilala ng Cake Pay!",
+  "cake_pay_learn_more": "Agan na bumili at mag-redeem ng mga gift card sa app!\nMag-swipe pakaliwa patungong kanan para matuto pa.",
+  "automatic": "Awtomatiko",
+  "fixed_pair_not_supported": "ang nakapirming pares na ito ay hindi sinusuportahan ng mga napiling palitan",
+  "variable_pair_not_supported": "Ang variable na pares na ito ay hindi sinusuportahan ng mga napiling palitan",
+  "none_of_selected_providers_can_exchange": "Wala sa mga napiling provider ang makakagawa ng palitan na ito",
+  "choose_one": "Pumili ng isa",
+  "choose_from_available_options": "Pumili mula sa mga variable na opsyon:",
+  "custom_redeem_amount": "Halaga ng custom na na-redeem",
+  "add_custom_redemption": "Magdagdag ng Custom Redemption",
+  "remaining": "Natitira",
+  "delete_wallet": "Tanggalin ang wallet",
+  "delete_wallet_confirm_message" : "Sigurado ka ba na gusto mong tanggalin ang iyong ${wallet_name} wallet?",
+  "low_fee": "Mababang bayad",
+  "low_fee_alert": "Kasalukuyan kang gumagamit ng mababang priyoridad ng bayad sa network. ito ay maaaring magdulot ng mahabang paghihintay, iba't ibang mga rate o nakanselang mga kalakalan. Inirerekomenda namin ang pagtatakda ng mas mataas na bayad para sa mas magandang karanasan.",
+  "ignor": "Huwag pansinin",
+  "use_suggested": "Iminumungkahing gamitin",
+  "do_not_share_warning_text" : "Huwag ibahagi ang mga ito sa sinuman kasama ang tagatustos.\n\nPwede o maaaring manakaw ang iyong pondo!",
```

Comment:
This is a very important warning and should sound scary enough to make people notice. No joke!

The second sentence is in English:

> Your funds can and will be stolen!

I propose to drop / not translate that complicated *can and will* construct and claim straight away:

> Maaaring manakaw ang iyong mga pondo!

or something similar.